File: typescript/fizzbuzz.ts
const myArray = Array.from({ length: 1e4 }, (_, i) => i + 1) as (string | number)[];

for (let i = 2; i < 1e4; i += 3) { myArray[i] = "Fizz"; }
for (let i = 4; i < 1e4; i += 5) { myArray[i] = "Buzz"; }
for (let i = 14; i < 1e4; i += 15) { myArray[i] = "FizzBuzz"; }

console.log(myArray);

export { };

File: python/fizzbuzz.py
def fizzBuzz():
        myArray = [i for i in range(1, 10001)]

        for i in range(2, 10000, 3): myArray[i] = "Fizz"
        for i in range(4, 10000, 5): myArray[i] = "Buzz"
        for i in range(14, 10000, 15): myArray[i] = "FizzBuzz"

        print(myArray)

fizzBuzz()
